### PR TITLE
chore(deps): use nodejs v14 version of aws jsii/superchain image

### PR DESCRIPTION
**Pull request by ansgarm**
This is mainly to test whether everything would still build and compile with this version.

We require Nodejs version 12 in our installation instructions but currently use the default jsii/superchain Docker image for builds which contains Nodejs v10. Unfortunately there is no Docker container with Nodejs v12 readily available for the jsii/superchain image.
The reason for this test is that the graphql-server, which is used in #801, needs Nodejs v12.

### CODE REVIEW

**Comment by ansgarm** [comment]
Hm, I just noticed that this might not work, we use a published version of our Docker container in our build and no version is published until we don't merge this into main 😅🤔

**Review by DanielMSchmidt — APPROVED** [review]

**Pull request merged**
